manifests/gateways/istio-ingress/templates/deployment.yaml

```diff
@@ -136,7 +136,7 @@ spec:
           {{- else if .Values.global.configNamespace }}
```

Comment:
why do we have three different variables for namespace? `global.istioNamespace`, `global.configNamespace` shouldn't be different than `.Release.Namespace`, right?

Reply:
Its an artifact of the passed. We should just have .Release.Namespace now. I have been meaning to clean it up but have not yet

Reply:
I see, thank you

Reply:
Please use istioNamespace - we want to reduce the use of Release.Namespace ( and other helm-specific features).
Even if we continue to use and support helm - we still want to avoid getting tied to its complexity and specific features. 

For example you can install an Ingress Gateway in a namespace 'foo' - but it still needs to set istioNamespace to istio-system to find the discovery server.